libs/nx-serverless/src/utils/packagers/index.ts

```diff
@@ -64,121 +26,127 @@ export function preparePackageJson(
   stats: any,
   resolverName: string,
   tsconfig?: string
-): Observable<BuilderOutput> {
+): Promise<BuilderOutput> {
+  const packageJsonPath = join(options.package, 'package.json');
+  const cwd = dirname(packageJsonPath);
   const resolver = resolverFactory(resolverName, context);
   context.logger.info('getting external modules');
   const workspacePackageJsonPath = join(context.workspaceRoot, 'package.json');
-  const packageJsonPath = join(options.package, 'package.json');
```

Comment:
why was this changed? The purpose is to get the minimal `package.json`

Reply:
It has not been changed, if you check line 30 its still there. If you ask why i moved it, i dont have a good reason :)